org.eclipse.scanning.api/src/org/eclipse/scanning/api/annotation/ui/FieldValue.java

```diff
@@ -300,9 +300,9 @@ public static boolean isModelField(Object model, String name) throws NoSuchField
 	 * @return fields old value, or null
 	 */
 	private static Object set(Object model, String name, Object value)throws IllegalAccessException, IllegalArgumentException, InvocationTargetException {
-		
+
 		Object oldValue = get(model, name);
-		
+
 		final String setter = getSetterName(name).toLowerCase();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "getSetterName" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)